### PR TITLE
Update PHP docs regarding changes to the `ErrorHandler`

### PR DESCRIPTION
**Pull request by stayallive**
## Pre-merge checklist

*If you work at Sentry, you're able to merge your own PR without review, but please don't unless there's a good reason.*

- [ ] Checked Vercel preview for correctness, including links
- [ ] PR was reviewed and approved by any necessary SMEs
- [ ] PR was reviewed and approved by a member of the [Sentry docs team](https://github.com/orgs/getsentry/teams/docs)

## Description of changes

In https://github.com/getsentry/sentry-php/pull/1633 we are changing how the PHP `ErrorHandler` works to correctly handle out of memory errors. These docs changes reflect those changes.

/cc @cleptric

### CODE REVIEW

**Review by cleptric — APPROVED** [review]

File: src/platforms/php/common/integrations/index.mdx [hunk 1]
```diff
@@ -23,7 +23,16 @@ To do so, it ensures that Sentry's `ErrorHandler` is registered, and adds a call
 
 This integration hooks into the global PHP `error_handler` and emits events when an error occurs.
 
-To do so, it ensures that Sentry's `ErrorHandler` is registered, and adds a callback to it as an error listener. By default, the `ErrorHandler` reserves 10 kilobytes of memory to handle fatal errors.
+To do so, it ensures that Sentry's `ErrorHandler` is registered, and adds a callback to it as an error listener. By default, the `ErrorHandler` reserves 16 KiB of memory to handle fatal errors. When handling out of memory errors the `ErrorHandler` additionally increases the [`memory_limit`](https://www.php.net/manual/en/ini.core.php#ini.memory-limit) by an additional 5 MiB to have enough room to send the out of memory event to Sentry, this change is only done once and only has effect for the handling of the out of memory error.
```
Comment (shanamatthews) [inline src/platforms/php/common/integrations/index.mdx:26]:
```suggestion
To do so, it ensures that Sentry's `ErrorHandler` is registered, and adds a callback to it as an error listener. By default, the `ErrorHandler` reserves 16 KiB of memory to handle fatal errors. When handling out of memory errors, the `ErrorHandler` additionally increases the [`memory_limit`](https://www.php.net/manual/en/ini.core.php#ini.memory-limit) by an additional 5 MiB to have enough room to send the out of memory event to Sentry. This change is only done once and only affects the handling of the out of memory error.
```

File: src/platforms/php/common/integrations/index.mdx [hunk 2]
```diff
@@ -23,7 +23,16 @@ To do so, it ensures that Sentry's `ErrorHandler` is registered, and adds a call
...
+
+If you want to change the amount of reserved memory for fatal error handling or the amount the `memory_limit` is increased with when handling an out of memory error you can register the fatal error handler yourself before calling `\Sentry\init(...)`:
```
Comment (shanamatthews) [inline src/platforms/php/common/integrations/index.mdx:28]:
```suggestion
To change the amount of memory reserved for fatal error handling or the amount the `memory_limit` is increased when handling an out of memory error, you can register the fatal error handler yourself before calling `\Sentry\init(...)`:
```

**Review by shanamatthews — APPROVED** [review]
Just some small punctuation tweaks to make this more readable. @stayallive -  do you want to check that these tweaks look correct to you? Then I'll merge the PR.

**Comment by stayallive** [comment]
Excellent changes, thanks 💪

**Pull request merged**